### PR TITLE
Added VLMC

### LINKED ISSUE #967: [NTR] vascular leptomeningeal cell
Note - please check that the term does not already exist (check OLS: https://www.ebi.ac.uk/ols/ontologies/cl)

**Preferred term label:**
Vascular leptomeningeal like cell

**Synonyms**
VLMC
Vascular leptomeningeal cell

**Definition (free text, please give PubMed ID in format PMID:XXXXXX)**
A neurohypophyseal cell that expresses Pdgfra and Lum. They are fibroblast-esque cells that are attached to vessels, located in the space betwee smooth muscle and astrocytes. Vascular leptomeningeal cells (VLMCs) like cells are a family of functionally related (but organ-specific) mesothelial fibroblasts that form protective membranes around internal organs, including the pia and arachnoid membranes of the brain. The fact that ENMFBs were obtained by sorting Wnt1-Cre;R26Tomato cells indicates that these cells are derived from the neural crest, as has previously been shown for both pia and arachnoid.
xref: ILX:0770143 

**Parent cell type term (check the hierarchy here https://www.ebi.ac.uk/ols/ontologies/cl):**
neuron associated cell? 

**What is the anatomical structure that the cell is a part of? Please check Uberon: https://www.ebi.ac.uk/ols/ontologies/uberon**

**Your nano-attribution (ORCID)**

**Any additional notes or concerns**
copied directly from ILX:0770143

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes #967

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -27772,26 +27780,6 @@ AnnotationAssertion(rdfs:label obo:D96882F1-8709-49AB-BCA9-772A67EA6C33 "alobate
 EquivalentClasses(obo:D96882F1-8709-49AB-BCA9-772A67EA6C33 ObjectIntersectionOf(obo:GO_0005634 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0002506)))
 SubClassOf(obo:D96882F1-8709-49AB-BCA9-772A67EA6C33 obo:GO_0005634)
 
-# Class: obo:PR_000013502 (parvalbumin alpha)
```
Comment (shawntanzk) [inline src/ontology/cl-edit.owl:27775]: Not quite sure what happened here, but basically there was extra labels to their synonym names, so I cleared it all so it would revert back to PR import names.

**Comment by shawntanzk** [comment]
Will change it to be under mesothelial fibroblast cell (need to I'll need to read up more before adding a term in CL for this)

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -27702,6 +27707,29 @@ SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P32848))
 
 
+# Class: obo:CL_4023051 (vascular leptomeningeal like cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "ILX:0770143") obo:IAO_0000115 obo:CL_4023051 "A neurohypophyseal cell that expresses Pdgfra and Lum. They are fibroblast-esque cells that are attached to vessels, located in the space betwee smooth muscle and astrocytes. Vascular leptomeningeal cells (VLMCs) like cells are a family of functionally related (but organ-specific) mesothelial fibroblasts that form protective membranes around internal organs, including the pia and arachnoid membranes of the brain.")
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:27712]: This definition looks  much more specific & detailed than necessary.  I'm not sure we need the markers at all to define it.  How about:
"A type of mesothelial fibroblast that is derived from the neural crest and is a key component of the pia and arachnoid membranes surrounding the brain."

Add: 
subClassOf part_of  some UBERON_0000391 # leptomeninx

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -27702,6 +27707,29 @@ SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
...
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023051 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023051 "ILX:0770143")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023051 "VLMC")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023051 "vascular leptomeningeal cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID: 30096314") rdfs:comment obo:CL_4023051 "The fact that ENMFBs were obtained by sorting Wnt1-Cre;R26Tomato cells indicates that these cells are derived from the neural crest, as has previously been shown for both pia and arachnoid.")
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:27717]: This needs unpacking a bit if it is do go into CL + we would need the original reference.  
Presumably 'ENMFB' = this cell type?  If this is evidence for the NC origin of these cells then that should be stated up front with a bit more detail about the experiment being referred to.

It's hard for me to tell from the detail provided whether this statement would make more sense in the Brain Data Standards Ontology as  a detail of mapping single cell data to these cell types.

**Review by dosumis — CHANGES_REQUESTED** [review]
See comments

File: src/ontology/cl-edit.owl (hunk 2 above)
Comment (dosumis) [reply src/ontology/cl-edit.owl:27712]: (BTW - interested to know why this is being called a 'neurohypophyseal cell' and what that might mean for functional assertions, & other potential axes of classification.  Would need a paper with evidence to back it up though.)

Comment (dosumis) [reply src/ontology/cl-edit.owl:27712]: Maybe chase down VLMC refs for makers from https://www.nature.com/articles/s41586-018-0654-5  - could have some other details.

**Review by dosumis — CHANGES_REQUESTED** [review]
Looks much better.  

Remaining issues:

- You need to put references directly on the definition and comments
- References used on the subClassOf axioms looks like they might be the wrong PMIDs. (https://pubmed.ncbi.nlm.nih.gov/5221728/;   https://pubmed.ncbi.nlm.nih.gov/5323293
 - I'm wondering about the 'like' in the name.  Do we need it. Is this just hedging language used because of some uncertainty about the mapping of transcriptomics clusters?


Sorry this one has been a bit of a saga.  I failed to spot all the issues at first.  I think we're nearly there.

**Comment by shawntanzk** [comment]
> You need to put references directly on the definition and comments
as appose to the subclass axiom? I've put reference at subclass axiom as the description does not mention the markers. Anyway with next update, I've added a PMID for definition (was ILX before as I copied and pasted the def directly) 

> References used on the subClassOf axioms looks like they might be the wrong PMIDs. (https://pubmed.ncbi.nlm.nih.gov/5221728/; https://pubmed.ncbi.nlm.nih.gov/5323293

whoops, sorry, my bad, immediately knew what it was - accidentally copied the PMCID instead of the PMID, fixed it. 

> I'm wondering about the 'like' in the name. Do we need it? Is this just hedging language used because of some uncertainty about the mapping of transcriptomics clusters?

Definitely need it in my opinion as the mapping is presumptive. The papers I've read tend to use the word "likely" when they describe this cluster (see [PMID:30096314](https://pubmed.ncbi.nlm.nih.gov/30096314/) for example) and from there they tend to use VLMC-like rather than VLMC)

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -27848,6 +27880,30 @@ SubClassOf(obo:CL_4023050 obo:CL_4023040)
...
 
+
+# Class: obo:CL_4023051 (vascular leptomeningeal like cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30096314"^^xsd:string) obo:IAO_0000115 obo:CL_4023051 "A type of mesothelial fibroblast that is derived from the neural crest and is a key component of the pia and arachnoid membranes surrounding the brain.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023051 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023051 "ILX:0770143")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023051 "VLMC")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023051 "vascular leptomeningeal cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:20023653"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:30096314"^^xsd:string) rdfs:comment obo:CL_4023051 "These enteric mesothelial fibroblasts (ENMFBs) are derived from the neural crest as evident from obtaining them by sorting Wnt1-Cre;R26Tomato cells (Tomato florescence labelled neural crest derived cells).")
```
[the remark this answers is not in the document]
Comment (dosumis) [reply src/ontology/cl-edit.owl:27891]: This is referring to a related cell type in a completely different location: "The fourth VLMC-like cell type (enteric mesothelial fibroblasts; ENMFBs) expressed all the VLMC marker genes but was found exclusively in the _enteric nervous system_." [PMID:20023653](https://www.cell.com/cell/fulltext/S0092-8674(18)30789-X?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS009286741830789X%3Fshowall%3Dtrue). (Note - not PMID:20023653 which is just about generation of markers. Refs should be sufficient for readers to find the source of assertions - as in any academic text.)

Looking over PMID:20023653 I think we probably need a general, species neutral term for mesothelial fibroblast of the leptomeninx (UBERON_0000391 = pia + arachnoid).  This would cover VLMC and other cell types mentioned in PMID:20023653 , e.g. ABC.  We should have species neutral terms for VLMC and ABC.  We can then look at adding species-specific terms with mouse markers and discuss whether we need a VLMC-like grouping class with explicit transcriptomics based def. 

Does that make sense?

**Review by dosumis — CHANGES_REQUESTED** [review]

File: src/ontology/cl-edit.owl (hunk 4 above)
[the remark this answers is not in the document]
Comment (shawntanzk) [reply src/ontology/cl-edit.owl:27891]: oh that makes more sense, i was wondering about that, thought it was some derived inference sort of situation. 
Currently they are species neutral, but I guess I should not add the markers in given that they were from mouse data. I've already added mesothelial fibroblast, but will add one specific to the leptomeninx under it. Happy to add a non species specific without the marker information, and add makers in another species specific entry. I'll also find better references. 

Things to do:

- [ ] Remove comment 
- [ ] Create mesothelial fibroblast of the leptomeninx & organise accordingly
- [ ] Remove markers from non species-specific term 
- [ ] Create species-specific entries and add above markers to them instead
- [ ] Fix references

[the remark this answers is not in the document]
Comment (shawntanzk) [reply src/ontology/cl-edit.owl:27891]: @dosumis think I fixed it - would like a review of it. Thanks

File: src/ontology/cl-edit.owl [hunk 5]
```diff
@@ -27884,6 +27925,45 @@ AnnotationAssertion(rdfs:label obo:CL_4023053 "spinal interneuron syanpsing Betz
...
+AnnotationAssertion(rdfs:label obo:CL_4023056 "vascular leptomeningeal cell (Mus musculus)")
+SubClassOf(obo:CL_4023056 obo:CL_4023051)
+SubClassOf(obo:CL_4023056 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_10090))
+SubClassOf(Annotation(oboInOwl:hasDbXref "PMID:30096314"^^xsd:string) obo:CL_4023056 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P11087))
+SubClassOf(Annotation(oboInOwl:hasDbXref "PMID:30096314"^^xsd:string) obo:CL_4023056 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P26618))
+SubClassOf(Annotation(oboInOwl:hasDbXref "PMID:30096314"^^xsd:string) obo:CL_4023056 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P51885))
+
+# Class: obo:CL_4023057 (arachnoid barrier cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:23298861") obo:IAO_0000115 obo:CL_4023057 "a cell that makes up the tight-junctioned layer of the leptomeninges that functions as the physiologic barrier between the CSF in the subarachnoid space and the fenestrated capillaries in the dura.")
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:27952]: Make part of arachnoid mater?  Also simplify def and make sure it doesn't duplicate def of tissue it is part of.  TO push this through, split out ABC to separate PR.

**Review by dosumis — CHANGES_REQUESTED** [review]

**Review by dosumis — APPROVED** [review]

File: src/ontology/cl-edit.owl [hunk 6]
```diff
@@ -27896,6 +27946,27 @@ SubClassOf(obo:CL_4023055 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070019))
...
+SubClassOf(Annotation(oboInOwl:hasDbXref "PMID:30096314"^^xsd:string) obo:CL_4023056 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P26618))
+SubClassOf(Annotation(oboInOwl:hasDbXref "PMID:30096314"^^xsd:string) obo:CL_4023056 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P51885))
+
+# Class: obo:CL_4023058 (mesothelial fibroblast of the leptomeninx)
+
+AnnotationAssertion(obo:IAO_0000115 obo:CL_4023058 "A mesothelial fibroblast found in the leptomeninx.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023058 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023058 "mesothelial fibroblast of the leptomeninx")
+SubClassOf(obo:CL_4023058 obo:CL_4023054)
+SubClassOf(obo:CL_4023058 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000391))
```
Comment (dosumis) [inline src/ontology/cl-edit.owl:27968]: Make this equivalentTo => autoclassification of of VLMC rather than asserted.

**Review by dosumis — CHANGES_REQUESTED** [review]
One more change (sorry).

**Review by patrick-lloyd-ray — APPROVED** [review]

**Comment by dosumis** [comment]
It's been bothering me that there's nothing in the definition of VLMC that distinguished them from the parent class we've created for them, from ABCs, or to account for the vascular in the name:

I checked [Marques](https://science.sciencemag.org/content/352/6291/1326.full)  and noticed

image

Needs to be reflected in text def.

**Comment by shawntanzk** [comment]
@dosumis yeah didn't notice but you are right. I've added "is localized in blood vessels" to the def and from the bits of reading, seems enough to distinguish them. Should I also add an axiom for that (maybe "located in" some "blood vessel")?

**Comment by dosumis** [comment]
Could add `'adjacent to' some 'blood vessel'` - but that might be a bit strong. I would say 'localized on' or 'associated with' in the text.

**Comment by shawntanzk** [comment]
> Could add `'adjacent to' some 'blood vessel'` - but that might be a bit strong. I would say 'localized on' or 'associated with' in the text.

I guess if its a bit strong we shouldn't add an axiom and just leave it in defs then. In which case should be all good now. Thanks

**Review by dosumis — APPROVED** [review]
